cmd/kubelet/app/server_test.go

```diff
@@ -0,0 +1,59 @@
+package app
+
+import (
+	"k8s.io/kubernetes/pkg/util/config"
+	"testing"
+)
+
+func TestValueOfAllocatableResources(t *testing.T) {
+	testCases := []struct {
+		kubeReserved   string
+		systemReserved string
+		errorExpected  bool
+		title          string
+	}{
+		// negative quantity value
```

Comment:
I'd recommend adding a `name` field to the struct and setting it to what you have in each item's comment, and then below, instead of logging the test case index, log the name instead.